cli/cmd/get.go

```diff
@@ -480,14 +489,73 @@ func describeAPI(name string, resourcesRes *schema.GetResourcesResponse) (string
 		sort.Strings(samplePlaceholderFields)
 		samplesPlaceholderStr := `{ "samples": [ { ` + strings.Join(samplePlaceholderFields, ", ") + " } ] }"
 		out += "Payload:  " + samplesPlaceholderStr + "\n"
-	}
-	if api != nil {
-		out += titleStr("Configuration") + api.UserConfigStr()
+	} else {
+		out += titleStr("Model Input Signature")
+
+		if groupStatus.Available() == 0 {
+			out += "Model "
+			return out, nil
+		}
+
+		modelInput, err := getModelInput(urls.Join(apiEndpoint, "signature"))
+		if err != nil {
+			return "", errors.Wrap(err, "")
```

Comment:
We might want to silently error here, since this might happen - right now I think "ready" is a little bit of a misnomer since the pod might be ready, but the server might not be running yet (although I haven't confirmed that yet, it's on my todo list)